packages/backend/src/accounting/tigerbeetle/accounts.ts

```diff
@@ -4,23 +4,30 @@ import {
   CreateAccountError as CreateAccountErrorCode
 } from 'tigerbeetle-node'
 
-import { ServiceDependencies, TigerbeetleAccountCode } from './service'
+import { ServiceDependencies, TigerBeetleAccountCode } from './service'
 import { TigerbeetleCreateAccountError } from './errors'
-import { AccountId, toTigerbeetleId } from './utils'
+import { AccountId, toTigerBeetleId, AccountUserData128 } from './utils'
 
 export interface CreateAccountOptions {
   id: AccountId
   ledger: number
-  code: TigerbeetleAccountCode
+  code: TigerBeetleAccountCode
+  // This flag links the result of this account creation to the result of the next one in the request,
+  // such that they will either succeed or fail together.
+  // The last account in a chain of linked accounts does not have this flag set.
+  linked?: boolean
+  // When set, the account will retain the history of balances at each transfer.
+  history?: boolean
+  assetId: AccountUserData128
```

Comment:
Should we get rid of `assetId` here? Or use it? Not sure what the intention was but it looks like it's unused here.

Reply:
@BlairCurrey The one in the PSQL wasn't used (I removed it now), but we want to use the `assetId` in order to have the same assetId when there is a liquidity/settlement account being created for an asset.
Also, good catch. I did not properly set it on the account. I have now. 
```
    accounts.map((account) => ({
      id: toTigerBeetleId(account.id),
      user_data_32: 0,
      user_data_64: 0n,
      user_data_128: toTigerBeetleId(account.assetId),
      reserved: 0,
      ledger: account.ledger,
      code: account.code,
      // Credit and debit accounts can both send and receive
      // but are restricted by their respective TigerBeetle flags.
      // In Rafiki transfers:
      // - the source account's debits increase
      // - the destination account's credits increase
      flags: flagsBasedOnAccountOptions(account),
      debits_pending: 0n,
      debits_posted: 0n,
      credits_pending: 0n,
      credits_posted: 0n,
      timestamp: 0n
    }))
```